### ISSUE TITLE
NoMethodError in 0.0.8

### ISSUE THREAD

**Issue by proby** [russ/sidekiq-history#13]
Hi, I was just trying out version 0.0.8 and ran into the following:
`NoMethodError: undefined method `[]' for nil:NilClass`

at
`gems/sidekiq-history-0.0.8/lib/sidekiq/history/middleware.rb:12:in `call'`

which is this line:
https://github.com/russ/sidekiq-history/blob/59e98f4495078b82e54eff7c6f7123328e58beef/lib/sidekiq/history/middleware.rb#L12

added in this commit:
https://github.com/russ/sidekiq-history/commit/23a53ff87c828c4ad262a904609858f3615ca4b5

For me, at that line `msg` is a single job data body:
```
{
  "class"=>"MyWorker",
  "args"=>[],
  "retry"=>true,
  "queue"=>"default",
  "jid"=>"85d4e0ff2cd4b5329796dede",
  "created_at"=>1493230018.898298,
  "enqueued_at"=>1493230067.107661,
  "error_message"=>"undefined method `[]' for nil:NilClass",
  "error_class"=>"NoMethodError",
  "failed_at"=>1493230018.901885,
  "retry_count"=>-1
}
```
So performing `msg['args'][0]['job_class']` on the above `Hash` will result in the `NoMethodError` because `msg['args'][0]` is `nil`. The fix might be to just do `msg['class']`.

I'm using the following sidekiq and sidekiq related gems on ruby 2.3.1 and rails 4.2.8:
* sidekiq (4.2.10)
* sidekiq-history (0.0.8)
* sidekiq-pro (3.4.5)
* sidekiq-scheduler (2.1.3)
* sidekiq-statistic (1.2.0)

**Comment by russ** [comment]
Good catch. I've got your recommend fix in place and release 0.0.9

Thanks!

**Comment by Fustrate** [comment]
@russ Looks like in my case, I need the original line because `msg['class']` is `ActiveJob::QueueAdapters::SidekiqAdapter::JobWrapper` for every job. Maybe that line should be

    job_class = msg['args'][0] ? msg['args'][0]['job_class'] : msg['class']
